scripts/publish.sh

```diff
@@ -45,5 +45,24 @@ npm run build
 (cd build && zip -r extension.zip ./)
 
 # upload and publish extension
-curl -H "Authorization: Bearer ${access_token}" -H "x-goog-api-version: 2" -X PUT -T build/extension.zip "https://www.googleapis.com/upload/chromewebstore/v1.1/items/${EXTENSION_ID}"
-curl -H "Authorization: Bearer ${access_token}" -H "x-goog-api-version: 2" -H "Content-Length: 0" -X POST "https://www.googleapis.com/chromewebstore/v1.1/items/${EXTENSION_ID}/publish"
+response=`curl -H "Authorization: Bearer ${access_token}" -H "x-goog-api-version: 2" -X PUT -T build/extension.zip "https://www.googleapis.com/upload/chromewebstore/v1.1/items/${EXTENSION_ID}" 2>/dev/null`
+echo "$response"
+status=`echo "console.log($response.uploadState)" | node`
+
+if [[ $status != "SUCCESS" ]]
```

Comment:
Nit:  I think this should be `!==` (following [Google style](https://google.github.io/styleguide/jsguide.html#features-equality-checks)).

Reply:
This is a shell script, not JavaScript so `!==` isn't valid here.